kv/src/main/java/com/test/kv/interfaces/Storage.java

```diff
@@ -0,0 +1,33 @@
+package com.test.kv.interfaces;
+
+import com.test.kv.exceptions.KVKeyDoesNotExistsException;
+
+/**
+ * Interface for storage
+ */
+public interface Storage<KeyType, ValueType> {
+    /**
+     * Get method returns the value of user specified key in storage if it exists
+     *
+     * @throws KVKeyDoesNotExistsException when key is not presented in storage
+     */
+    ValueType Get(KeyType key) throws KVKeyDoesNotExistsException;
```

Comment:
since it throw exception on key missing, I think the interface need another .HasKey method to allow test key existence.